Week1/homework/js-exercises/compareArrays.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+//declare 2 array variable
+const schoolItems = ['books', 'pens', 'notebook', 'tablet'];
+const studentsInfo = [
+  { name: 'ahmed' },
+  { age: 13 },
+  { grad: 'A' },
+  { height: 120 },
+  { weight: 30 },
+  { country: 'palestine' },
+  { telehone: '06000000' },
+];
```

Comment:
In this case you created multiple objects in an array, normally you would have every key value pair within 1 object. Like so:

```
const studentsInfo = {
  name: 'ahmed',
  age: 13,
  grad: 'A',
  ...
};
```

Reply:
i have implement that in the new PR i hope i did it in the right way